torch/onnx/_internal/io_adapter.py

```diff
@@ -103,6 +225,27 @@ def apply(
         return tuple(model_args) + tuple(model_kwargs.values()), {}
 
 
+class AppendPositionalsIntoArgsStep:
+    """Merge the input kwargs into the input args."""
```

Comment:
docstring and below comments don't match.

Reply:
oopsy, someone copied and pasted code here